### ISSUE TITLE
Maybe add support for scala?

### ISSUE THREAD

**Issue by ptillemans** [trustin/sphinx-maven-plugin#3]
In order to support mixed java/scala projects, it would be nice to include the sphinxcontrib.scaladomain package in the dist folder.

**Comment by trustin** [comment]
Thanks for suggestion. Didn't know there is Scala domain implementation. https://pythonhosted.org/sphinxcontrib-scaladomain/ Will try to include in the next release.

**Comment by trustin** [comment]
Not sure why, but the installation fails, even if I install docutils explicitly. My guess is that scaladomain depends on an older version of docutil than what Sphinx uses. I tried to look into the source code repository at: https://bitbucket.org/birkenfeld/sphinx-contrib/src but scaladomain did not exist.

```
Searching for sphinxcontrib-scaladomain==0.1a1
Reading https://pypi.python.org/simple/sphinxcontrib-scaladomain/
Downloading https://pypi.python.org/packages/16/69/0523e707fb23873a752d9a6a2c6d5d1c1f010d5dfccc1d528e50d10a20dd/sphinxcontrib-scaladomain-0.1a1.tar.gz#md5=41f34ec8d4e4e3deabafada1b66e133f
Best match: sphinxcontrib-scaladomain 0.1a1
Processing sphinxcontrib-scaladomain-0.1a1.tar.gz
Writing /tmp/easy_install-M_hFQK/sphinxcontrib-scaladomain-0.1a1/setup.cfg
Running sphinxcontrib-scaladomain-0.1a1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-M_hFQK/sphinxcontrib-scaladomain-0.1a1/egg-dist-tmp-6arddd
Traceback (most recent call last):
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/bin/easy_install", line 11, in <module>
    sys.exit(main())
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/command/easy_install.py", line 2261, in main
    setup(
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/command/easy_install.py", line 410, in run
    self.easy_install(spec, not self.no_deps)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/command/easy_install.py", line 665, in easy_install
    return self.install_item(spec, dist.location, tmpdir, deps)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/command/easy_install.py", line 695, in install_item
    dists = self.install_eggs(spec, download, tmpdir)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/command/easy_install.py", line 876, in install_eggs
    return self.build_and_install(setup_script, setup_base)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/command/easy_install.py", line 1115, in build_and_install
    self.run_setup(setup_script, setup_base, args)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/command/easy_install.py", line 1101, in run_setup
    run_setup(setup_script, args)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/command/easy_install.py", line 1101, in run_setup
    run_setup(setup_script, args)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/sandbox.py", line 248, in run_setup
    DirectorySandbox(setup_dir).run(runner)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/sandbox.py", line 198, in setup_context
    yield
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/sandbox.py", line 169, in save_modules
    saved_exc.resume()
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/sandbox.py", line 144, in resume
    six.reraise(type, exc, self._tb)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/sandbox.py", line 248, in run_setup
    DirectorySandbox(setup_dir).run(runner)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/sandbox.py", line 248, in run_setup
    DirectorySandbox(setup_dir).run(runner)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/sandbox.py", line 278, in run
    return func()
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/sandbox.py", line 246, in runner
    _execfile(setup_script, ns)
  File "/home/trustin/Workspaces/sphinx-maven-plugin/target/update-sphinx/jython/Lib/site-packages/setuptools/sandbox.py", line 47, in _execfile
    exec(code, globals, locals)
  File "/tmp/easy_install-M_hFQK/sphinxcontrib-scaladomain-0.1a1/setup.py", line 4, in <module>
  File "/tmp/easy_install-M_hFQK/sphinxcontrib-scaladomain-0.1a1/sphinxcontrib/scaladomain.py", line 15, in <module>
ImportError: No module named docutils
```

**Comment by trustin** [comment]
Hmm, splitting easy_install into two phases seems to work, i.e. `easy_install docutils && easy_install sphinxcontrib-scaladomain` works while `easy_install docutils sphinxcontrib-scaladomain` doesn't. Maybe because `EGG-INFO/requires.txt` is incorrect?

**Comment by trustin** [comment]
Fixed by e9bd8a819b64f78075aa15ea648a0c34eba52424

**Comment by trustin** [comment]
@ptillemans Just releases sphinx-maven-plugin 1.6.2. Tried scaladomain briefly and it seems like it generates some warning. I'm not sure it is really in usable state to be honest. It's just a single file which doesn't harm anyway. :-)
